tests/unit/test_lexer.py

```diff
@@ -119,6 +119,18 @@ def testBlockBlankLines(self):
         tokens = self.lexer.tokenize(text)
         self.assertEqual(tokens, ['a', '\n\n\n', 'a', '\n'])
 
+    def testSelfClosingTags(self):
+        text = "<block/>"
+        tokens = self.lexer.tokenize(text)
+        self.assertEqual(tokens, ['block'])
+
+    def testSelfClosingTagsDisabled(self):
+        options = { 'disableselfclosingtags': True }
+        text = "<block/>\n<block />\n<block hello/>"
```

Comment:
Just wondering, would `"""` block of text be more readable/manageable than a single line with embedded `\n`? ;-)